server/src/main/java/fr/uga/l3miage/tp1/exo3/models/BrandMiage.java

```diff
@@ -0,0 +1,23 @@
+package fr.uga.l3miage.tp1.exo3.models;
+
+import fr.uga.l3miage.tp1.exo3.enums.BrandType;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "brand_miage")
+public class BrandMiage {
```

Comment:
les noms des classes doit être suffixé par `Entity`  
```suggestion
public class BrandMiageEntity {
```